example/package.json

```diff
@@ -17,15 +17,17 @@
     "react-native-config": "^1.5.0",
     "react-native-device-info": "^10.3.0",
     "react-native-keychain": "^8.1.1",
-    "react-query": "^3.39.3"
+    "react-native-webview": "^11.26.1",
+    "react-query": "^3.39.3",
+    "reactotron-react-native": "^5.0.3"
```

Comment:
Once I added this (for debugging a 403), it was going to be annoying to disentangle from the rest of this.  I figured few enough changes no one would mind.